src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,104 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private List<Entry<K, V>>[] table;
+    private int size;
+
+    public MyHashMap() {
+        this.table = new ArrayList[INITIAL_CAPACITY];
+        this.size = 0;
+    }
 
     @Override
     public void put(K key, V value) {
-
+        int hash = hash(key);
+        int index = getIndex(hash);
```

Comment:
refactor `getIndex` method accordingly
```suggestion
        int index = getIndex(key);
```